horace_core/admin/check_horace_mex.m

```diff
@@ -1,17 +1,29 @@
-function [rez, n_errors, can_use_mex_4_combine] = check_horace_mex()
+function [rez, n_errors, can_use_mex_4_combine,can_use_MPI] = check_horace_mex()
 % function checks if horace mex files are compiled correctly and returns
 % their version string.
 %
 % Usage:
 %   >>[rez, n_errors] = check_horace_mex();
-%   >>[rez, n_errors, can_use_mex_4_combine] = check_horace_mex();
+%   >>[rez, n_errors, can_use_mex_4_combine,can_use_herbert_mpi] = check_horace_mex();
```

Comment:
unclear why herbert is appearing here in the description

Reply:
because parallel computing toolbox uses Matlab MPI and Herbert may use Poor man MPI  which is not an MPI but pretends to be such (writes/reads/deletes files pretending to be MPI messages). And may use `cpp_communicator` which supports proper MPI implementation compiled for Herbert